assets/js/mojFrontendInit.js

```diff
@@ -0,0 +1 @@
+window.MOJFrontend.initAll()
```

Comment:
Have you put this in a `script` tag in your html layout?

Reply:
Yup.